docs/source/getting-started/installation.rst

```diff
@@ -7,47 +7,35 @@ Installing the SDK
 1. Install the dependencies
 ***************************
 
-The SDK currently runs on Windows, MacOS or Linux.
+The SDK currently runs on Windows, macOS and Linux.
 
 You need to install:
 
 1. `Visual Studio Code <https://code.visualstudio.com/download>`_.
-2. JDK 8 or greater.
+2. JDK 8 or greater. Any JDK will do, but if you don't have a preference, here is a link to `AdoptOpenJDK <https://adoptopenjdk.net>`_.
```

Comment:
```suggestion
2. JDK 8 or greater. If you don’t already have a JDK installed, try `AdoptOpenJDK <https://adoptopenjdk.net>`_.
```
Users that care or know the difference between various JDKs, will install their favorite JDK anyway and for users that don’t I think pointing them at something very directly is helpful.

Reply:
👍